extract-types.mjs

```diff
@@ -20,9 +20,20 @@ function extractTypes(srcDir, destDir) {
     if (entry.isDirectory()) {
       extractTypes(srcPath, destPath);
     } else if (entry.isFile() && entry.name.endsWith('.d.ts')) {
-      fs.copyFileSync(srcPath, destPath);
+      console.log(destPath);
```

Comment:
Do we need this still?

Reply:
I thought it was  nice to leave in, kind of as a sanity check. Otherwise, you could run this, it could do –nothing– and you'd do a `git diff` and think nothing was changed.

Reply:
happy to remove if you'd like

Reply:
Nah you can leave it. I'm just used to myself forgetting to delete them.